extensions/wikia/InsightsBlogpostRedirect/InsightsBlogpostRedirectController.class.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+class InsightsBlogpostRedirectController extends WikiaSpecialPageController {
+
+	public function __construct() {
+		parent::__construct( 'Insights', 'insights', true );
+	}
+
+	public function index() {
+		$this->wg->Out->redirect( wfMessage('insights-blogpost-url')->escaped() );
```

Comment:
I'd use `->useDatabase( false )` here. We shouldn't let admins modify a URL we redirect a user to.

Also, `wfMessage` needs to be spacey per conventions.